fta/samples/models.py

```diff
@@ -97,14 +97,23 @@ class LabeledSample(models.Model):
 
     objects = LabeledSampleManager()
 
-    original_sample = models.OneToOneField(
-        Sample,
+    original_sample = models.ForeignKey(
+        to=Sample,
         related_name="labeled_sample",
         on_delete=models.CASCADE,
         blank=False,
         null=False,
         help_text="Link to the original sample",
     )
+
+    superseded_by = models.ForeignKey(
+        to="self",
+        related_name="labeled_sample",
```

Comment:
I don't think `'labeled_sample'` is a good `related_name` here, maybe `'superseded_labeled_samples'` or something like that.

Reply:
ok